### PR TITLE
Update geometry for SFO 

### PR DESCRIPTION
**Pull request by thisisaaronland**
This PR updates the geometry for SFO to use CCSF/SFO-sourced data. The previous USZCTA geometry has been moved in to an alternate geometry. CCSF/SFO-sourced data has the following changes:

* Removal of the "the leaf" which is not SFO property but a SAMTRANS bus depot.
* Removal of the US Coast Guard station (since it's not technically SFO property).
* Removal of US 101 which bisects the airport property
* Improved (accurate) geometry for the airport's West-of-Bayshore wetlands property
* Improved coastline geometry
 
Screen Shot 2021-07-14 at 13 48 24
Screen Shot 2021-07-14 at 14 15 39

### CODE REVIEW

**Comment by thisisaaronland** [comment]
Update: I have noticed some extraneous (bunk) data caused during the clipping phase in the principal campus geometry. I will update this shortly.

File: data/102/527/513/102527513.geojson [hunk 1]
```diff
@@ -182,9 +182,10 @@
     ],
     "oa:elevation_ft":"13",
     "oa:type":"large_airport",
-    "src:geom":"uszcta",
+    "src:geom":"ccsf-sfo",
```
Comment (nvkelso) [inline data/102/527/513/102527513.geojson:185]: Confirmed we already have this source here: https://github.com/whosonfirst/whosonfirst-sources/blob/50557d12f81b871d9c76de826a85d193a07b96cd/sources/ccsf-sfo.json.

**Review by nvkelso — APPROVED** [review]
Geom changes look good to me!

**Comment by thisisaaronland** [comment]
Now with an updated geometry that removes extraneous (incorrect) land-mass in front of the SFO Marriott.

Screen Shot 2021-07-14 at 15 17 12

File: data/102/527/513/102527513.geojson [hunk 2]
```diff
@@ -182,9 +182,10 @@
...
     "src:geom_alt":[
-        "woedb"
+        "woedb",
+        "uszcta"
```
Comment (stepps00) [inline data/102/527/513/102527513.geojson:188]: @thisisaaronland possible to add "uszcta" to the `"wof:geom_alt"`property as well? Other than that, this PR looks good.

**Comment by stepps00** [comment]
One small note about the `"wof:geom_alt"`, but looks good otherwise :+1:

**Comment by thisisaaronland** [comment]
@stepps00 - Updated. Do we even need to keep `src:alt_geom` ?

**Comment by stepps00** [comment]
> @stepps00 - Updated. Do we even need to keep src:alt_geom ?

That's been the standard since the new `wof:alt_geom` property was introduced (more info about it in https://github.com/whosonfirst-data/whosonfirst-data/issues/1793).. I am open to discussing it further, though.

**Pull request merged**